server/main.go

```diff
@@ -27,20 +29,32 @@ func main() {
 	// init your devices here
 	d := domain.Devices{
 		Temperature: &devices.Temperature{},
+		Wind:        &devices.Wind{},
 	}
 
 	s := &domain.IoTService{DB: db, Devices: &d}
 	//storage.Storage = db
 	router := s.NewRouter()
 
-	s.DB.AutoMigrate(devices.TemperatureEvent{}, devices.Sensor{})
+	s.DB.AutoMigrate(devices.WindEvent{}, devices.TemperatureEvent{}, devices.Sensor{})
 	// prepare device
 	err = s.Devices.Temperature.CreateSensor()
 	if err != nil {
 		log.Fatal(fmt.Printf("Error creating device: %v \n", err))
 		return
 	}
 
+	err = s.Devices.Wind.CreateSensor()
+	if err != nil {
+		log.Fatal(fmt.Printf("Error creating device: %v \n", err))
+		return
+	}
+
+	c := cors.New(cors.Options{
```

Comment:
Then remove `headers setting` in your handlers if it uses wildcard and methods here